datahub-web-react/src/app/ingest/source/builder/DefineRecipeStep.tsx

```diff
@@ -46,15 +46,21 @@ export const DefineRecipeStep = ({ state, updateState, goTo, prev }: StepProps)
 
     const { type } = state;
     const sourceConfigs = getSourceConfigs(type as string);
+    const isEditing: boolean = prev === undefined;
     const displayRecipe = stagedRecipeYml || sourceConfigs.placeholderRecipe;
     const sourceDisplayName = sourceConfigs.displayName;
     const sourceDocumentationUrl = sourceConfigs.docsUrl; // Maybe undefined (in case of "custom")
 
+    // LookML banner specific code
+    const isSourceLooker: boolean = sourceConfigs.type === 'looker' || false;
```

Comment:
this can just be `const isSourceLooker: boolean = sourceConfigs.type === 'looker'` no || false needed

Reply:
Fixed!